model_compression_toolkit/core/common/quantization/quantization_params_fn_selection.py

```diff
@@ -40,23 +40,10 @@ def get_activation_quantization_params_fn(activation_quantization_method: Quanti
 
     """
     if activation_quantization_method == QuantizationMethod.POWER_OF_TWO:
-        # Use min/max as the threshold if we use NOCLIPPING
         if activation_error_method == QuantizationErrorMethod.NOCLIPPING:
```

Comment:
Why NOCLIPPING in not inside power_of_two_selection_histogram ? (same for weights)